src/main/java/com/redhat/devtools/intellij/commonUiTestLibrary/fixtures/mainIdeWindow/menuBar/MenuBar.java

```diff
@@ -0,0 +1,83 @@
+/*******************************************************************************
+ * Copyright (c) 2021 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution,
+ * and is available at https://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package com.redhat.devtools.intellij.commonUiTestLibrary.fixtures.mainIdeWindow.menuBar;
+
+import com.intellij.remoterobot.RemoteRobot;
+import com.intellij.remoterobot.fixtures.ComponentFixture;
+import com.intellij.remoterobot.fixtures.ContainerFixture;
+
+import java.time.Duration;
+import java.util.List;
+
+import static com.intellij.remoterobot.search.locators.Locators.byXpath;
+import static com.redhat.devtools.intellij.commonUiTestLibrary.UITestRunner.getIntelliJIdeaVersion;
+
+/**
+ * Top menu fixture
+ *
+ * @author zcervink@redhat.com
+ */
+public class MenuBar {
+    private RemoteRobot remoteRobot;
+    private final int ideaVersion = getIntelliJIdeaVersion();
```

Comment:
in FlatWelcomeFrame it is set in constructor. Why you don't use the same approach everywhere?

Reply:
Fixed.